contracts/instruments/RibbonVolatility.sol

```diff
@@ -312,18 +326,22 @@ contract RibbonVolatility is DSMath, InstrumentStorageV1, InstrumentStorageV2 {
                 adapter,
                 optionType,
                 amount,
-                strikePrice
+                strikePrice,
+                paymentToken,
+                maxCost
             );
         } else if (purchaseMethod == PurchaseMethod.ZeroEx) {
-            purchaseWithZeroEx(adapter, optionType, strikePrice, buyData);
+            purchaseWithZeroEx(adapter, optionType, strikePrice, buyData, paymentToken);
```

Comment:
I understand the intent here to future-proof, but I don't really see the us using a payment token for 0x purchases anytime soon. Most Opyn buys are using USDC. Can just remove the `paymentToken` param for `purchaseWithZeroEx`

Reply:
ok done